rollup/internal/controller/relayer/l2_relayer.go

```diff
@@ -403,6 +404,12 @@ func (r *Layer2Relayer) ProcessPendingBatches() {
 
 		txHash, err := r.commitSender.SendTransaction(dbBatch.Hash, &r.cfg.RollupContractAddress, calldata, blob, fallbackGasLimit)
 		if err != nil {
+			if errors.Is(err, sender.ErrTooManyPendingBlobTxs) {
+				r.metrics.rollupL2RelayerProcessPendingBatchErrTooManyPendingBlobTxsTotal.Inc()
+				log.Debug("Didn't send commitBatch tx to layer 1",
+					"err", err)
```

Comment:
```suggestion
				log.Debug("Skipped sending commitBatch tx to L1: too many pending blob txs", "maxPending", r.cfg.SenderConfig.MaxPendingBlobTxs, "err", err,)
				
```

Reply:
refined some wording, feel free to tweak it, the error's msg is descriptive, btw.